gpu-jobs.md

```diff
@@ -55,8 +67,45 @@ capabilities that are available to any CHTC user (as of 2/7/2020):
     <td>yes</td>
   </tr>
   <tr>
-    <td>4</td>
-    <td>gpu2002 - gpu2005</td>
+    <td>2</td>
+    <td>gpu2002,gpu2003</td>
+    <td>8</td>
+    <td>GeForce RTX 2080 Ti</td>
+    <td>CentOS 7</td>
+    <td>yes</td>
+  </tr>
+</table>
+
+### Job types, runtimes, and per-user limitations
+
+{:.gtable}
+  | Job type | Submit file name (?) | Maximum runtime | Per-user limitation |
+  | --- |
+  | Short | ? | 12 hrs | 2/3 of CTHC GPU Lab GPUs |
+  | Medium | ? | 24 hrs | 1/3 of CTHC GPU Lab GPUs |
+  | Long | ? | 7 days | 1 job with 1-2 GPUs |
+  | Interactive | ? | 4 hours | 1 job with 1 GPU |
```

Comment:
In a GPU Lab team meeting today, we decided to modify the table.  `Interactive` should remain in the table but is not  a job type.  The `Submit file name (?)` column can be generalized because it pertains to how to run that type of job.

We also proposed a `condor_submit` wrapper called `condor_submit_interactive` that replaces `condor_submit -i` (https://htcondor-wiki.cs.wisc.edu/index.cgi/tktview?tn=7552).  However, that will be a medium term update.  We plan to merge these updates first with the existing `condor_submit -i`  instructions for interactive jobs.